syntax = "proto3";

package chord;

service ClientService {
    rpc Insert(InsertRequest) returns (InsertResponse) {}
    rpc Delete(DeleteRequest) returns(DeleteResponse)  {}
    rpc Query(QueryRequest) returns(QueryResponse) {}
}

message InsertRequest {
    string song = 1;
    string value=2;
}

message InsertResponse {
    string response = 1;
}

message DeleteRequest {
    string song = 1;
}

message DeleteResponse{
    string response =1;
}

message QueryRequest{
    string song =1;
}

message QueryResponse{
    string value =1;
  
}